test/extensions/filters/http/ext_proc/streaming_integration_test.cc

```diff
@@ -36,7 +36,8 @@ class StreamingIntegrationTest : public HttpIntegrationTest,
 
 protected:
   StreamingIntegrationTest() : HttpIntegrationTest(Http::CodecType::HTTP2, ipVersion()) {}
-  ~StreamingIntegrationTest() { TearDown(); }
+  // TODO(yanjunxiang-google): Verify that bypassing virtual dispatch here was intentional
```

Comment:
cc @yanjunxiang-google

Reply:
Hmmm, I suggest to add NOLINT and TODO here. This type update require to be very careful.